caps/management/commands/import_filter_data.py

```diff
@@ -0,0 +1,68 @@
+import os
+import pandas as pd
+
+import requests
+import urllib3
+
+import ssl
+
+ssl._create_default_https_context = ssl._create_unverified_context
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+from django.core.management.base import BaseCommand, CommandError
+from django.conf import settings
+
+from os.path import join
+
+from scoring.models import PlanScore
+from caps.models import PlanDocument
+
+FILTER_DATA_URL = "https://raw.githubusercontent.com/mysociety/climate_league_filter_datasets/main/data/outputs/all_filters.csv"
+FILTER_CSV = join(settings.DATA_DIR, "all_filters.csv")
+
+
+def get_data():
+    token = os.environ.get("PERSONAL_ACCESS_TOKEN", None)
+    headers = {"Authorization": "token " + token}
+    r = requests.get(FILTER_DATA_URL, headers=headers)
+    with open(FILTER_CSV, "wb") as outfile:
+        outfile.write(r.content)
+
+
+def import_filters():
+    ruc_lookup = dict((desc, code) for code, desc in PlanScore.RUC_TYPES)
+
+    updates = []
+    df = pd.read_csv(FILTER_CSV)
+    df["council-quintile"] = df["council-quintile"].fillna(0).astype(int)
+    df["pop_bucket"] = df["pop_bucket"].fillna("")
+    df["political_control"] = df["political_control"].fillna("")
+
+    for index, row in df.iterrows():
+        plan_score = PlanScore.objects.get(year=2021, council__authority_code=row["local-authority-code"])
```

Comment:
In other cases I think you referred to a year variable in settings? But also, not a blocker.